extensions/websockets-next/runtime/src/main/java/io/quarkus/websockets/next/WebSocketConnector.java

```diff
@@ -94,6 +95,16 @@ default WebSocketConnector<CLIENT> baseUri(String baseUri) {
      */
     WebSocketConnector<CLIENT> addSubprotocol(String value);
 
+    /**
+     * Add a value to the connection user data.
+     *
+     * @param key
+     * @param value
+     * @param <VALUE>
+     * @return self
+     */
```

Comment:
Let's add a link to `WebSocketClientConnection#userData()`?

Reply:
Good idea. I added a link to `UserData#put(TypedKey, Object)` also.